assets/js/components/ReportError.js

```diff
@@ -119,14 +118,7 @@ export default function ReportError( { moduleSlug, error } ) {
 		</Fragment>
 	);
 
-	const retryableErrors = errors.filter(
-		( err ) =>
-			!! err?.selectorData?.storeName &&
-			err.selectorData?.name === 'getReport' &&
-			! isInsufficientPermissionsError( err ) &&
-			! isPermissionScopeError( err ) &&
-			! isAuthError( err )
-	);
+	const retryableErrors = errors.filter( ( err ) => isErrorRetryable( err ) );
```

Comment:
This line can be simplified:

```suggestion
	const retryableErrors = errors.filter( isErrorRetryable );
```